File: C#/Linear-Search.cs
using System;

namespace LinearSearch {
  class Program {
    public static int linear(int[] array, int key) {
      for (int i = 0; i<array.Length; i++) {
        if (array[i] == key) return i;
      }
      return -1;
    }
    
    static void Main() {
      int[] array = new int[] {5, 3, 7, 44, 63, 53, 35, 36, 9, 45};
      Console.WriteLine(linear(array, 3));
    }
  }
}

File: Java/Linear-Search.java
public class Main {
  public static int linear(int[] array, int key) {
    for (int i = 0; i<array.length; i++) {
      if (array[i] == key) {
        return i;
      }
    }
    return -1; //not found
  }
  
  public static void main(String[] args) {
    int[] a = new int[] {5, 2, 6, 2, 9, 45, 635, 4, 747, 35, 3};
    System.out.println(linear(a, 2));
  }
}
